week-1-practise/level-1/9-function-output-2.js

```diff
@@ -1,14 +1,20 @@
-function s(w1, w2) {
-    return w1.concat(w2);
+function s(w1, w2, w3) {
+  return w1.concat(w2, w3);
 }
 
-var result = s('code', 'your');
+var result = s("code", "your", "Future");
 console.log(result);
 
-// Answer these questions:
+// Answer these questions.
 // - What does this program do? Run it and see, or Google some of the methods used ot understand them.
+// .concat(); chaines (concatenate) the string arguments to the calling strings and returns a new string
 // - How many parameters does the function take?
+// The funtion takes 2 parameters.
 // - What is the function name?
+// the function name is s.
 // - Where is the function called? with what parameters?
+// The function is called at line 5 which is s, and the parameters are called with the following arguments : code, your, future.
 // - What does the function return?
-// - How can this function be improved?
\ No newline at end of file
+// the function return "codeyour".
+// - How can this function be improved?
+// this function can be improved strings with space inside the parenthese of concat.
```

Comment:
You could also improve the names of the variables to make it easier to read. Otherwise, good answers.